dnf5/include/dnf5/context.hpp

```diff
@@ -37,6 +40,25 @@ class Plugins;
 
 constexpr const char * VERSION = "0.1.0";
 
+
+struct LoadPackage {
+    std::string url;
+    std::optional<libdnf::rpm::Package> package;
+};
```

Comment:
What is the purpose of this structure? Because I do not see it used anywhere in this PR.

Reply:
I made more version of code. This is remmant of one of them. I will remove the code.